src/main/java/frc/robot/Robot.java

```diff
@@ -736,6 +804,15 @@ public void turretLoop() {
   }
 
   public void driveTrainLoop() {
+
+    // Hack
+    if (mAuto) {
```

Comment:
Why not conditionally execute `driveTrainLoop()` where auto is executing it instead of executing then immediately returning?

Reply:
We are doing this now. I wasn't done cleaning up this branch when max decided to merge it.